ts/src/orderbook.ts

```diff
@@ -119,7 +119,8 @@ export const orderBook = {
                 assetPair.assetDataA.assetData === assetData || assetPair.assetDataB.assetData === assetData;
             nonPaginatedFilteredAssetPairs = assetPairsItems.filter(containsAssetData);
         }
-        const paginatedFilteredAssetPairs = paginate(nonPaginatedFilteredAssetPairs, page, perPage);
+        const uniqueNonPaginatedFilteredAssetPairs = _.uniqWith(nonPaginatedFilteredAssetPairs, _.isEqual);
```

Comment:
```suggestion
        const uniqueNonPaginatedFilteredAssetPairs = _.uniqWith(nonPaginatedFilteredAssetPairs, _.isEqual.bind(_));
```
I know it's weird but otherwise linter fails.